src/fides/cli/commands/system_scanner.py

```diff
@@ -0,0 +1,181 @@
+"""Defines the CLI commands for the System Scanner"""
+
+from os import path
+from subprocess import DEVNULL, CalledProcessError, run
+from typing import Any, Callable, List, Tuple
+
+from click import Context, echo, group, option, pass_context
+
+from ...ctl.core.utils import echo_green, echo_red
+from ..options import dry_flag, verbose_flag, yes_flag
+from ..utils import with_analytics
+
+RESERVED_LABELS = [
+    "app",
+    "component",
+    "vizier-bootstrap",
+    "vizier-updater-dep",
+]
+
+
+def run_px(command: List[str]) -> None:
+    px_command = " ".join(["px"] + command)
+    run(px_command, check=True, shell=True, stderr=DEVNULL)
+
+
+def with_px_auth(func: Callable) -> Callable:
+    def wrapper(ctx: Context, *args, **kwargs) -> Any:  # type: ignore
+        if "system_scanner" not in ctx.obj["CONFIG"]:
+            echo_red("System scanner configuration not found")
+            raise SystemExit(1)
+
+        if "pixie_api_key" not in ctx.obj["CONFIG"].system_scanner:
+            echo_red("System scanner API key not found")
+            raise SystemExit(1)
```

Comment:
Same as my above comment. I'm unsure how to test this. If I comment these out, the below `px auth login` command succeeds within the container, when I substitute my personal API key for the variable.